examples/examples.py

```diff
@@ -0,0 +1,86 @@
+# # ddsp-drum examples
+#
+# We walk through basic functionality of `ddsp-drum` in this Jupyter notebook.
+#
+# Just note that all ipd.Audio play widgets normalize the audio.
+
+# +
+# %load_ext autoreload
+# %autoreload 2
+# %matplotlib inline
+
+import numpy as np
+import matplotlib.pyplot as plt
+import os
+from ddspdrum.module import ADSR, VCO, VCA
+from scipy.signal import stft as stft
+from scipy.io import wavfile as wavfile
+import librosa.display
+import IPython.display as ipd
+# -
+
+# Synthesis parameters.
+a = 0.1
+d = 0.1
+s = 0.5
+r = 0.5
+alpha = 3
+sustain_for = 0.25
+
+f0 = 440
+
+# TODO: Why don't we add 's' here?
+dur = a + d + r + sustain_for
+
+# Envelope test
+adsr = ADSR(a, d, s, r, alpha)
+# TODO: Can you explain the envelope here?
+env = adsr(sustain_for)
+
+# Let's avoid cryptic variables names when possible
```

Comment:
I really want to avoid cryptic names, particularly because our audience might include people who don't have a DSP background.

Reply:
The only names I could see as cryptic:

env --> envelope

t_cr --> timeline at control rate
t_sr --> timeline at sample rate

Reply:
yes. just more a note moving forward